lib/cumulus/ifupdown2.rb

```diff
@@ -151,6 +151,17 @@ def update_speed
     @confighash['config']['link-duplex'] = 'full'
   end
 
+  def update_updown_scripts
```

Comment:
Т.к. переменная называется `updown`, предлагаю этот метод переименовать по аналогии с другими: `update_<variable>`:
```suggestion
  def update_updown
```